wiki/Community/Forum/Modding_Queues/en.md

```diff
@@ -0,0 +1,20 @@
+---
+stub: true
+---
+
+# Modding Queues
+
+<!-- TODO: this needs to be linked from other related articles, for example, "how to get mods" -->
+
+[**Modding Queues**](https://osu.ppy.sh/community/forums/60) is a forum dedicated to [beatmap review](/wiki/Modding) requests, or mods. It consists of forum threads, or *queues* belonging to different modders and mappers. Every queue has individual requirements and limits on obtaining the mods, which must be respected by its visitors in order to receive a mod.
+
+## Queue rounds
+
+Before the start of every round, the queue's owner posts a short announcement, which typically contains:
```

Comment:
this needs a sentence before it explaining what a "round" is and why most queues operate that way